src/main_menu.h

```diff
@@ -5,11 +5,13 @@
 #include <cstddef>
 #include <string>
 #include <vector>
+#include <ctime>
```

Comment:
Why do you have this include? You don't appear to need it here.

Reply:
Looks leftover from a previous iteration.  Will remove.